Middleware/SVGAnimations/Opacity.cs

```diff
@@ -0,0 +1,21 @@
+using System;
+
+public class Opacity
```

Comment:
You're using `public` everywhere. Some of that may be valid, but it's a very prevalent bad habit in C# code. You should use `internal` by default and only public where you're exposing types to another project (or sometimes to the framework itself, eg your middleware might need to stay public).